test/e2e/fixture/scylla/scylladbcluster.yaml.tmpl

```diff
@@ -34,6 +34,8 @@ spec:
       nodes: 1
     racks:
     - name: a
```

Comment:
nit: use more descriptive names. At least rack0, rack1, rack2 or something along these lines.

Reply:
I don't see any added value here. If it's good enough for our docs, it's good enough for tests.